modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -6,17 +6,61 @@
 [id="lvms-upgrading-lvms-on-sno_{context}"]
 = Upgrading {lvms} on {sno} clusters
 
-Currently, it is not possible to upgrade from {rh-storage} Logical Volume Manager Operator 4.11 to {lvms} 4.12 on {sno} clusters.
+You can upgrade the {lvms-first} Operator to align with your {sno} version.
 
-[IMPORTANT]
-====
-The data will not be preserved during this process.
-====
+// Is data preserved now?
+
+.Prerequisites
```

Comment:
To reduce wordiness, you could either put the "You have:" as an intro sentence before the bullet points with them rewritten as, for example, "Upgraded your cluster".  Alternatively, you could leave off the intro sentence and still use the abbreviated bullet points.

Reply:
I see you point. We use these phrases a lot in our rrepo in prereqs:
You have installed the OpenShift CLI (`oc`)" a lot in our repo.
You have installed the OpenShift CLI (`oc`).

For that reason I'll just keep the same structure in this instance